aws/resource_aws_kms_ciphertext.go

```diff
@@ -0,0 +1,85 @@
+package aws
+
+import (
+	"encoding/base64"
+	"log"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/kms"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsKmsCiphertext() *schema.Resource {
+
+	return &schema.Resource{
+		Create: resourceAwsKmsCiphertextCreate,
+		Read:   resourceAwsKmsCiphertextRead,
+		Delete: resourceAwsKmsCiphertextDelete,
+
+		Schema: map[string]*schema.Schema{
+			"plaintext": {
```

Comment:
It might be worth specifying `Sensitive: true,` here: https://github.com/hashicorp/terraform/blob/86e6481cc661b2ba694d353aab6d11838c066c7c/helper/schema/schema.go#L200

Reply:
Good, thanks fixing.